Bonsai.Tinkerforge/BrickletGPSV2.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Tinkerforge;
+
+namespace Bonsai.Tinkerforge
+{
+    [DefaultProperty(nameof(Uid))]
+    [Description("Measures coordinates, altitude, modtion, timestamp, satellite status from a GPS Bricklet 2.0.")]
```

Comment:
Typo in motion, but do we actually measure motion, or just the GPS coordinates? Maybe we revise this to just say something like "Measures GPS coordinates, altitude, satellite status and timer data?".

Also it might be easier to use if the order of the data in the tuple matches the documentation, or if we create a helper class like DataFrame where each data is named.

Finally, see below for a suggestion on different implementation approach.

Reply:
There is definitely a reference to motion data in the documentation, gives us course and speed